app/src/main/java/io/github/droidkaigi/confsched2017/view/fragment/SessionsFragment.java

```diff
@@ -116,6 +121,8 @@ public void onResume() {
     @Override
     public void onStop() {
         super.onStop();
+        int position = binding.recyclerView.getFirstVisiblePosition();
+        sharedPreferences.edit().putInt("last_position", position).apply();
```

Comment:
Actually this app uses `kvs-schema`.
So you can use auto generated method to access shared preferences.
Can you follow the steps below?

## 1. Add key to `DefaultPrefsSchema`

```java
@Table(name = "io.github.droidkaigi.confsched_preferences")
public interface DefaultPrefsSchema {
    @Key(name = "last_sessions_view_position")
    int lastSessionsViewPosition;
}
```

## 2. Use generated method.
```java
DefaultPrefs prefs = DefaultPrefs.get(context);
int lastPosition = prefs.getLastSessionsViewPosition();
```